### DISCUSSION TITLE
How to enable GraphQL on older Supabase account

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#7255] [Questions] [answered]
GraphQL is available on all newer instances, but mine is quite old. I hear you can install it yourself, but no idea how. 
Any insight appreciated.
Thanks

**Comment by silentworks** [comment, marked as answer]
I don't think older instance will have this extension. I just tried to enable it on one of my old instances and it said the extension cannot be found. If you are on a paid plan you can open a support request https://app.supabase.io/support/new otherwise you can create a new project which will have GraphQL and migrate your old project over https://supabase.com/docs/guides/database#migrating-between-projects.

**Reply by jtkeyva** [reply]
Thank you!
